extractor/src/main/java/rs/valence/extractor/extractors/Blocks.java

```diff
@@ -37,6 +39,13 @@ public JsonElement extract() {
             blockJson.addProperty("translation_key", block.getTranslationKey());
             blockJson.addProperty("item_id", Registries.ITEM.getRawId(block.asItem()));
 
+            if (block.asItem() instanceof VerticallyAttachableBlockItem wsbItem) {
+                if (wsbItem.getBlock() == block) {
+                    var wallBlock = ((ExposeWallBlock) wsbItem).getWallBlock();
+                    blockJson.addProperty("wall_variant_id", Registries.BLOCK.getRawId(wallBlock));
```

Comment:
I'm not sure whether to call it wall variant or something from `VerticallyAttachableBlockItem`.

In the original commit, the name of the import was `WallStandingBlockItem`, but this didn't work so I had to change this - did the mappings change?

Reply:
There might have been some tweaks in a later version of the yarn mappings.

Reply:
Actually `wall_block_id` is probably a better name for this.

Reply:
Renamed the public function, although I've left the internals alone.